firefox_puppeteer/api/appinfo.py

```diff
@@ -0,0 +1,21 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class AppInfo(BaseLib):
+
+    @property
+    def browserTabsRemoteAutostart(self):
```

Comment:
I wouldn't call it like the preference. If there is a name change, we would have to update a lot of tests maybe. Lets call it `multi_process`, `e10s_enabled`, or simply `e10s`.